### ISSUE TITLE
Bundle install with bundler 1.11.x errors with "no implicit conversion of Array into String"

### ISSUE THREAD

**Issue by jnixon22** [bundler/bundler#4275]
This seems to work with 1.10.x.  
I ran the command `/usr/local/bin/bundle install --standalone`
  
And got:

```
Error details
    TypeError: no implicit conversion of Array into String
      /usr/local/lib/ruby/gems/2.0.0/gems/bundler-1.11.2/lib/bundler/installer/standalone.rb:44:in `initialize'
      /usr/local/lib/ruby/gems/2.0.0/gems/bundler-1.11.2/lib/bundler/installer/standalone.rb:44:in `new'
      /usr/local/lib/ruby/gems/2.0.0/gems/bundler-1.11.2/lib/bundler/installer/standalone.rb:44:in `gem_path'
      /usr/local/lib/ruby/gems/2.0.0/gems/bundler-1.11.2/lib/bundler/installer/standalone.rb:29:in `block (2 levels) in paths'
      /usr/local/lib/ruby/gems/2.0.0/gems/bundler-1.11.2/lib/bundler/installer/standalone.rb:28:in `map'
      /usr/local/lib/ruby/gems/2.0.0/gems/bundler-1.11.2/lib/bundler/installer/standalone.rb:28:in `block in paths'
      /usr/local/lib/ruby/gems/2.0.0/gems/bundler-1.11.2/lib/bundler/installer/standalone.rb:26:in `map'
      /usr/local/lib/ruby/gems/2.0.0/gems/bundler-1.11.2/lib/bundler/installer/standalone.rb:26:in `paths'
      /usr/local/lib/ruby/gems/2.0.0/gems/bundler-1.11.2/lib/bundler/installer/standalone.rb:17:in `block in generate'
      /usr/local/lib/ruby/gems/2.0.0/gems/bundler-1.11.2/lib/bundler/installer/standalone.rb:11:in `open'
      /usr/local/lib/ruby/gems/2.0.0/gems/bundler-1.11.2/lib/bundler/installer/standalone.rb:11:in `generate'
      /usr/local/lib/ruby/gems/2.0.0/gems/bundler-1.11.2/lib/bundler/installer.rb:73:in `run'
      /usr/local/lib/ruby/gems/2.0.0/gems/bundler-1.11.2/lib/bundler/installer.rb:21:in `install'
      /usr/local/lib/ruby/gems/2.0.0/gems/bundler-1.11.2/lib/bundler/cli/install.rb:105:in `run'
      /usr/local/lib/ruby/gems/2.0.0/gems/bundler-1.11.2/lib/bundler/cli.rb:172:in `install'
      /usr/local/lib/ruby/gems/2.0.0/gems/bundler-1.11.2/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
      /usr/local/lib/ruby/gems/2.0.0/gems/bundler-1.11.2/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
      /usr/local/lib/ruby/gems/2.0.0/gems/bundler-1.11.2/lib/bundler/vendor/thor/lib/thor.rb:359:in `dispatch'
      /usr/local/lib/ruby/gems/2.0.0/gems/bundler-1.11.2/lib/bundler/vendor/thor/lib/thor/base.rb:440:in `start'
      /usr/local/lib/ruby/gems/2.0.0/gems/bundler-1.11.2/lib/bundler/cli.rb:10:in `start'
      /usr/local/lib/ruby/gems/2.0.0/gems/bundler-1.11.2/exe/bundle:19:in `block in <top (required)>'
      /usr/local/lib/ruby/gems/2.0.0/gems/bundler-1.11.2/lib/bundler/friendly_errors.rb:7:in `with_friendly_errors'
      /usr/local/lib/ruby/gems/2.0.0/gems/bundler-1.11.2/exe/bundle:17:in `<top (required)>'
      /usr/local/bin/bundle:23:in `load'
      /usr/local/bin/bundle:23:in `<main>'
```

Environment
    Bundler   1.11.2
    Rubygems  2.0.14
    Ruby      2.0.0p481 (2014-05-08 revision 45883) [x86_64-linux]
    GEM_HOME  /opt/mfp/deploy/apps/active/Main/contents/vendor/bundle/ruby/2.0.0
    GEM_PATH  
    Git       1.9.1

Bundler settings
    clean
      Set for your local app (/opt/mfp/deploy/apps/active/Main/contents/.bundle/config): "true"
    without
      Set for your local app (/opt/mfp/deploy/apps/active/Main/contents/.bundle/config): "development:test"
    frozen
      Set for your local app (/opt/mfp/deploy/apps/active/Main/contents/.bundle/config): "1"
    path
      Set for your local app (/opt/mfp/deploy/apps/active/Main/contents/.bundle/config): "vendor/bundle"
    disable_shared_gems
      Set for your local app (/opt/mfp/deploy/apps/active/Main/contents/.bundle/config): "1"

**Comment by TimMoore** [comment]
Can you post your `Gemfile` please?

**Comment by jnixon22** [comment]
`source 'https://rubygems.org'

gem 'rdoc', '3.12'
gem 'Text', :require => ['text', 'text/metaphone']
gem 'active_presenter', '3.2.2'
gem 'acts_as_list', '0.3.0'
gem 'airbrake', '4.1.0'
gem 'ar_mailer_rails3', '2.1.12'
gem 'awesome_print', '1.2.0'
gem 'aws-s3', '0.6.3'
gem 'aws-ses', '0.4.4', :require => 'aws/ses'
gem 'bcrypt-ruby', '3.0.1'
gem 'bson_ext', '1.8.5'
gem 'calendar_date_select', '1.16.4'
gem 'clicktale', '1.1.0'
gem 'daemons', '1.1.8'
gem 'dalli', '2.1.0'
gem 'elasticsearch', '1.0.0'
gem 'elasticsearch-api', '1.0.0'
gem 'elasticsearch-transport', '1.0.0'
gem 'es_client', '0.0.6'
gem 'geoip', '1.1.2'
gem 'hpricot', '0.8.6'
gem 'http_accept_language', '2.0.1'
gem 'httparty', '0.11.0'
gem 'httpclient', '2.2.7'
gem 'jquery-rails', '2.0.2'
gem 'kgio', '2.9.2'
gem "koala", "1.11.1"
gem 'mechanize', '2.5.1'
gem 'mini_magick', '3.6.0'
gem 'mongo', '1.8.5'
gem 'msgpack', '0.4.7'
gem 'mysql2', '0.3.11'
gem 'net-sftp', '2.0.5'
gem 'omniauth', '1.1.4'
gem 'omniauth-twitter', '0.0.16'
gem 'passenger', '4.0.29'
gem 'patron', '0.4.18'
gem 'rails', '3.2.21'
gem 'rails-translate-routes', '0.1.3'
gem 'rake', '0.9.6'
gem 'recaptcha', '0.4.0', :require => 'recaptcha/rails'
gem 'redis', '3.0.4'
gem 'responders', '1.0.0'
gem 'resque', '1.23.0'
gem 'resque-retry', '1.0.0'
gem 'resque-scheduler', '2.2.0'
gem 'resque_mailer', '2.1.0'
gem 'retryable', '1.3.2'
gem 'rinku', '1.7.0', :require => 'rails_rinku'  # replacement for auto_link
gem 'ruby-units', '1.4.4'
gem 'rubypress', '1.0.1'
gem 'sanitize', '2.0.3'
gem 'sass-rails', '3.2.6'
gem 'scout', '5.5.5'
gem 'secure_headers', '1.0.0'
gem 'sendgrid-ruby', '0.0.3'
gem 'sort_alphabetical', '0.1.3'
gem 'statsd-ruby', '1.0.0', :require => 'statsd'
gem 'tinder', '1.9.1'
gem 'twitter', '4.8.1'
gem 'twitter-text', '1.6.1'
gem 'user-agent', '1.0.0'
gem 'will_paginate', '3.0.3'
gem 'timeout', '0.0.0'
gem 'auth-client', '1.0.8'
gem 'desk', '1.0.0'
gem 'active_model_serializers', '0.9.3'
gem 'strong_parameters'
gem 'rack-reverse-proxy', require: 'rack/reverse_proxy', :git => 'https://github.com/jaswope/rack-reverse-proxy.git', :tag => 'master'
gem 'message_bus_client', '1.0.4'
gem 'faraday', '0.8.9'
gem 'faraday_middleware', '0.9.0'
gem 'safe_attributes', '1.0.10'
gem 'tzinfo', '0.3.42'
gem 'rack-cors', :require => 'rack/cors'

group :assets do
  gem 'execjs', '2.2.2'
  gem 'therubyracer', '0.12.1'
  gem 'turbo-sprockets-rails3', '0.3.14'
  gem 'uglifier', '2.5.3'
  gem 'yui-compressor', '0.12.0'
  gem 'closure-compiler', '~> 1.1.11'
end

group :development do
  gem 'thin', '1.6.3'
  gem 'smartling'
  gem 'rails-dev-boost', :git => 'git://github.com/thedarkone/rails-dev-boost.git'
end

group :development, :test do
  gem 'factory_girl_rails', '3.5.0'
  gem 'rspec-rails', '2.14.2'
  gem 'rails-dev-tweaks', '~> 1.1'
end

group :test do
  gem 'capybara', '2.4.4'
  gem 'mock_redis', '0.6.0'
  gem 'rb-fsevent', '0.9.2', :require => false
  gem 'rb-inotify', '0.8.8', :require => false
  gem 'simplecov', '0.8.2'
  gem 'terminal-notifier-guard', '1.5.3', :require => false
  gem 'database_cleaner', '1.2.1'
  gem 'vcr'
end

group :production do
  gem 'newrelic_rpm', '3.7.2.192'
end
`

**Comment by TimMoore** [comment]
`gem 'es_client', '0.0.6'` doesn't seem to exist, so I can't get this to resolve at all https://rubygems.org/gems/es_client/versions

Can you post your `Gemfile.lock` as well please?

**Comment by TimMoore** [comment]
I also can't find `auth-client`: https://rubygems.org/search?utf8=%E2%9C%93&query=auth-client

**Comment by jnixon22** [comment]
Sorry, I tried to sweep through and remove our internal gems, but missed a few.  I can reproduce without those gems.  Here's the Gemfile.lock

``GIT
  remote: git://github.com/thedarkone/rails-dev-boost.git
  revision: a709a10b06b7d9fc4a9846bc613c60c86af9dd7c
  specs:
    rails-dev-boost (0.3.0)
      railties (>= 3.0)

GIT
  remote: https://github.com/jaswope/rack-reverse-proxy.git
  revision: 7b8304fb5fe5105667802da782ddcbd26ba8822f
  tag: master
  specs:
    rack-reverse-proxy (0.8.1)
      rack (>= 1.0.0)
      rack-proxy (~> 0.5)

GEM
  remote: https://rubygems.org/
  specs:
    Text (1.1.3)
      text (>= 1.2.0)
    actionmailer (3.2.21)
      actionpack (= 3.2.21)
      mail (~> 2.5.4)
    actionpack (3.2.21)
      activemodel (= 3.2.21)
      activesupport (= 3.2.21)
      builder (~> 3.0.0)
      erubis (~> 2.7.0)
      journey (~> 1.0.4)
      rack (~> 1.4.5)
      rack-cache (~> 1.2)
      rack-test (~> 0.6.1)
      sprockets (~> 2.2.1)
    active_model_serializers (0.9.3)
      activemodel (>= 3.2)
    active_presenter (3.2.2)
    activemodel (3.2.21)
      activesupport (= 3.2.21)
      builder (~> 3.0.0)
    activerecord (3.2.21)
      activemodel (= 3.2.21)
      activesupport (= 3.2.21)
      arel (~> 3.0.2)
      tzinfo (~> 0.3.29)
    activeresource (3.2.21)
      activemodel (= 3.2.21)
      activesupport (= 3.2.21)
    activesupport (3.2.21)
      i18n (~> 0.6, >= 0.6.4)
      multi_json (~> 1.0)
    acts_as_list (0.3.0)
      activerecord (>= 3.0)
    addressable (2.4.0)
    airbrake (4.1.0)
      builder
      multi_json
    ar_mailer_rails3 (2.1.12)
    arel (3.0.3)
    awesome_print (1.2.0)
    aws-s3 (0.6.3)
      builder
      mime-types
      xml-simple
    aws-ses (0.4.4)
      builder
      mail (> 2.2.5)
      mime-types
      xml-simple
    bcrypt-ruby (3.0.1)
    brwsr (1.1.1)
    bson (1.8.6)
    bson_ext (1.8.5)
      bson (~> 1.8.5)
    builder (3.0.4)
    calendar_date_select (1.16.4)
    capybara (2.4.4)
      mime-types (>= 1.16)
      nokogiri (>= 1.3.3)
      rack (>= 1.0.0)
      rack-test (>= 0.5.4)
      xpath (~> 2.0)
    clicktale (1.1.0)
    closure-compiler (1.1.12)
    daemon_controller (1.2.0)
    daemons (1.1.8)
    dalli (2.1.0)
    database_cleaner (1.2.1)
    desk (1.0.0)
      faraday (~> 0.8.0)
      faraday_middleware (~> 0.9.0)
      hashie (~> 1.2.0)
      json (~> 1.7)
      multi_json (~> 1.6)
      multi_xml (~> 0.5)
      pony (~> 1.1)
      rash (~> 0.3.1)
      simple_oauth (~> 0.2.0)
    diff-lcs (1.2.5)
    docile (1.1.5)
    domain_name (0.5.25)
      unf (>= 0.0.5, < 1.0.0)
    elasticsearch (1.0.0)
      elasticsearch-api (= 1.0.0)
      elasticsearch-transport (= 1.0.0)
    elasticsearch-api (1.0.0)
      multi_json
    elasticsearch-transport (1.0.0)
      faraday
      multi_json
    elif (0.1.0)
    erubis (2.7.0)
    eventmachine (1.0.9.1)
    execjs (2.2.2)
    factory_girl (3.5.0)
      activesupport (>= 3.0.0)
    factory_girl_rails (3.5.0)
      factory_girl (~> 3.5.0)
      railties (>= 3.0.0)
    faraday (0.8.9)
      multipart-post (~> 1.2.0)
    faraday_middleware (0.9.0)
      faraday (>= 0.7.4, < 0.9)
    ffi (1.9.10)
    geoip (1.1.2)
    hashie (1.2.0)
    hike (1.2.3)
    hpricot (0.8.6)
    http-cookie (1.0.2)
      domain_name (~> 0.5)
    http_accept_language (2.0.1)
    httparty (0.11.0)
      multi_json (~> 1.0)
      multi_xml (>= 0.5.2)
    httpclient (2.2.7)
    i18n (0.7.0)
    journey (1.0.4)
    jquery-rails (2.0.2)
      railties (>= 3.2.0, < 5.0)
      thor (~> 0.14)
    json (1.8.3)
    kgio (2.9.2)
    koala (1.11.1)
      addressable
      faraday
      multi_json
    libv8 (3.16.14.13)
    mail (2.5.4)
      mime-types (~> 1.16)
      treetop (~> 1.4.8)
    mechanize (2.5.1)
      domain_name (~> 0.5, >= 0.5.1)
      mime-types (~> 1.17, >= 1.17.2)
      net-http-digest_auth (~> 1.1, >= 1.1.1)
      net-http-persistent (~> 2.5, >= 2.5.2)
      nokogiri (~> 1.4)
      ntlm-http (~> 0.1, >= 0.1.1)
      webrobots (~> 0.0, >= 0.0.9)
    message_bus_client (1.0.4)
      activemodel (~> 3.2.21, >= 3.2.21)
    mime-types (1.25.1)
    mini_magick (3.6.0)
      subexec (~> 0.2.1)
    mock_redis (0.6.0)
    mongo (1.8.5)
      bson (~> 1.8.5)
    msgpack (0.4.7)
    multi_json (1.11.2)
    multi_xml (0.5.5)
    multipart-post (1.2.0)
    mysql2 (0.3.11)
    net-http-digest_auth (1.4)
    net-http-persistent (2.9.4)
    net-sftp (2.0.5)
      net-ssh (>= 2.0.9)
    net-ssh (2.9.2)
    netrc (0.11.0)
    newrelic_rpm (3.7.2.192)
    nokogiri (1.5.11)
    ntlm-http (0.1.1)
    oauth (0.4.7)
    omniauth (1.1.4)
      hashie (>= 1.2, < 3)
      rack
    omniauth-oauth (1.1.0)
      oauth
      omniauth (~> 1.0)
    omniauth-twitter (0.0.16)
      multi_json (~> 1.3)
      omniauth-oauth (~> 1.0)
    passenger (4.0.29)
      daemon_controller (>= 1.1.0)
      rack
      rake (>= 0.8.1)
    patron (0.4.18)
    polyglot (0.3.5)
    pony (1.11)
      mail (>= 2.0)
    rack (1.4.5)
    rack-cache (1.2)
      rack (>= 0.4)
    rack-cors (0.4.0)
    rack-protection (1.5.3)
      rack
    rack-proxy (0.5.17)
      rack
    rack-ssl (1.3.4)
      rack
    rack-test (0.6.3)
      rack (>= 1.0)
    rails (3.2.21)
      actionmailer (= 3.2.21)
      actionpack (= 3.2.21)
      activerecord (= 3.2.21)
      activeresource (= 3.2.21)
      activesupport (= 3.2.21)
      bundler (~> 1.0)
      railties (= 3.2.21)
    rails-dev-tweaks (1.2.0)
      actionpack (>= 3.1)
      railties (>= 3.1)
    rails-translate-routes (0.1.3)
      activesupport (>= 3.1)
    railties (3.2.21)
      actionpack (= 3.2.21)
      activesupport (= 3.2.21)
      rack-ssl (~> 1.3.2)
      rake (>= 0.8.7)
      rdoc (~> 3.4)
      thor (>= 0.14.6, < 2.0)
    rake (0.9.6)
    rash (0.3.2)
      hashie (~> 1.2.0)
    rb-fsevent (0.9.2)
    rb-inotify (0.8.8)
      ffi (>= 0.5.0)
    rdoc (3.12)
      json (~> 1.4)
    recaptcha (0.4.0)
    redis (3.0.4)
    redis-namespace (1.5.2)
      redis (~> 3.0, >= 3.0.4)
    ref (2.0.0)
    responders (1.0.0)
      railties (>= 3.2, < 5)
    resque (1.23.0)
      multi_json (~> 1.0)
      redis-namespace (~> 1.0)
      sinatra (>= 0.9.2)
      vegas (~> 0.1.2)
    resque-retry (1.0.0)
      resque (>= 1.10.0)
      resque-scheduler (>= 1.9.9)
    resque-scheduler (2.2.0)
      redis (>= 3.0.0)
      resque (>= 1.20.0, < 1.25)
      rufus-scheduler (~> 2.0)
    resque_mailer (2.1.0)
      actionmailer (~> 3.0)
    rest-client (1.8.0)
      http-cookie (>= 1.0.2, < 2.0)
      mime-types (>= 1.16, < 3.0)
      netrc (~> 0.7)
    retryable (1.3.2)
    rinku (1.7.0)
    roauth (0.0.8)
    rspec-core (2.14.8)
    rspec-expectations (2.14.5)
      diff-lcs (>= 1.1.3, < 2.0)
    rspec-mocks (2.14.6)
    rspec-rails (2.14.2)
      actionpack (>= 3.0)
      activemodel (>= 3.0)
      activesupport (>= 3.0)
      railties (>= 3.0)
      rspec-core (~> 2.14.0)
      rspec-expectations (~> 2.14.0)
      rspec-mocks (~> 2.14.0)
    ruby-units (1.4.4)
    rubypress (1.0.1)
    rufus-scheduler (2.0.24)
      tzinfo (>= 0.3.22)
    safe_attributes (1.0.10)
      activerecord (>= 3.0.0)
    sanitize (2.0.3)
      nokogiri (>= 1.4.4, < 1.6)
    sass (3.4.21)
    sass-rails (3.2.6)
      railties (~> 3.2.0)
      sass (>= 3.1.10)
      tilt (~> 1.3)
    scout (5.5.5)
      elif
    secure_headers (1.0.0)
      brwsr (>= 1.1.1)
    sendgrid-ruby (0.0.3)
      rest-client
      smtpapi
    simple_oauth (0.2.0)
    simplecov (0.8.2)
      docile (~> 1.1.0)
      multi_json
      simplecov-html (~> 0.8.0)
    simplecov-html (0.8.0)
    sinatra (1.4.6)
      rack (~> 1.4)
      rack-protection (~> 1.4)
      tilt (>= 1.3, < 3)
    smartling (1.0.1)
      multi_json (~> 1.0)
      rest-client (~> 1.6)
    smtpapi (0.1.0)
    sort_alphabetical (0.1.3)
      activesupport
    sprockets (2.2.3)
      hike (~> 1.2)
      multi_json (~> 1.0)
      rack (~> 1.0)
      tilt (~> 1.1, != 1.3.0)
    statsd-ruby (1.0.0)
    strong_parameters (0.2.3)
      actionpack (~> 3.0)
      activemodel (~> 3.0)
      activesupport (~> 3.0)
      railties (~> 3.0)
    subexec (0.2.3)
    terminal-notifier-guard (1.5.3)
    text (1.3.1)
    therubyracer (0.12.1)
      libv8 (~> 3.16.14.0)
      ref
    thin (1.6.3)
      daemons (~> 1.0, >= 1.0.9)
      eventmachine (~> 1.0)
      rack (~> 1.0)
    thor (0.19.1)
    tilt (1.4.1)
    timeout (0.0.0)
    tinder (1.9.1)
      eventmachine (>= 0.12.0, < 2)
      faraday (~> 0.8)
      faraday_middleware (~> 0.8)
      hashie (~> 1.0)
      json (~> 1.6)
      mime-types (~> 1.16)
      multi_json (~> 1.0)
      multipart-post (~> 1.1)
      twitter-stream (~> 0.1)
    treetop (1.4.15)
      polyglot
      polyglot (>= 0.3.1)
    turbo-sprockets-rails3 (0.3.14)
      railties (> 3.2.8, < 4.0.0)
      sprockets (>= 2.2.0)
    twitter (4.8.1)
      faraday (~> 0.8, < 0.10)
      multi_json (~> 1.0)
      simple_oauth (~> 0.2)
    twitter-stream (0.1.12)
      eventmachine (>= 0.12.8)
      roauth (>= 0.0.2)
    twitter-text (1.6.1)
      unf (~> 0.0.5)
    tzinfo (0.3.42)
    uglifier (2.5.3)
      execjs (>= 0.3.0)
      json (>= 1.8.0)
    unf (0.0.5)
      unf_ext
    unf_ext (0.0.7.1)
    user-agent (1.0.0)
    vcr (3.0.1)
    vegas (0.1.11)
      rack (>= 1.0.0)
    webrobots (0.1.2)
    will_paginate (3.0.3)
    xml-simple (1.1.5)
    xpath (2.0.0)
      nokogiri (~> 1.3)
    yui-compressor (0.12.0)

PLATFORMS
  ruby

DEPENDENCIES
  Text
  active_model_serializers (= 0.9.3)
  active_presenter (= 3.2.2)
  acts_as_list (= 0.3.0)
  airbrake (= 4.1.0)
  ar_mailer_rails3 (= 2.1.12)
  awesome_print (= 1.2.0)
  aws-s3 (= 0.6.3)
  aws-ses (= 0.4.4)
  bcrypt-ruby (= 3.0.1)
  bson_ext (= 1.8.5)
  calendar_date_select (= 1.16.4)
  capybara (= 2.4.4)
  clicktale (= 1.1.0)
  closure-compiler (~> 1.1.11)
  daemons (= 1.1.8)
  dalli (= 2.1.0)
  database_cleaner (= 1.2.1)
  desk (= 1.0.0)
  elasticsearch (= 1.0.0)
  elasticsearch-api (= 1.0.0)
  elasticsearch-transport (= 1.0.0)
  execjs (= 2.2.2)
  factory_girl_rails (= 3.5.0)
  faraday (= 0.8.9)
  faraday_middleware (= 0.9.0)
  geoip (= 1.1.2)
  hpricot (= 0.8.6)
  http_accept_language (= 2.0.1)
  httparty (= 0.11.0)
  httpclient (= 2.2.7)
  jquery-rails (= 2.0.2)
  kgio (= 2.9.2)
  koala (= 1.11.1)
  mechanize (= 2.5.1)
  message_bus_client (= 1.0.4)
  mini_magick (= 3.6.0)
  mock_redis (= 0.6.0)
  mongo (= 1.8.5)
  msgpack (= 0.4.7)
  mysql2 (= 0.3.11)
  net-sftp (= 2.0.5)
  newrelic_rpm (= 3.7.2.192)
  omniauth (= 1.1.4)
  omniauth-twitter (= 0.0.16)
  passenger (= 4.0.29)
  patron (= 0.4.18)
  rack-cors
  rack-reverse-proxy!
  rails (= 3.2.21)
  rails-dev-boost!
  rails-dev-tweaks (~> 1.1)
  rails-translate-routes (= 0.1.3)
  rake (= 0.9.6)
  rb-fsevent (= 0.9.2)
  rb-inotify (= 0.8.8)
  rdoc (= 3.12)
  recaptcha (= 0.4.0)
  redis (= 3.0.4)
  responders (= 1.0.0)
  resque (= 1.23.0)
  resque-retry (= 1.0.0)
  resque-scheduler (= 2.2.0)
  resque_mailer (= 2.1.0)
  retryable (= 1.3.2)
  rinku (= 1.7.0)
  rspec-rails (= 2.14.2)
  ruby-units (= 1.4.4)
  rubypress (= 1.0.1)
  safe_attributes (= 1.0.10)
  sanitize (= 2.0.3)
  sass-rails (= 3.2.6)
  scout (= 5.5.5)
  secure_headers (= 1.0.0)
  sendgrid-ruby (= 0.0.3)
  simplecov (= 0.8.2)
  smartling
  sort_alphabetical (= 0.1.3)
  statsd-ruby (= 1.0.0)
  strong_parameters
  terminal-notifier-guard (= 1.5.3)
  therubyracer (= 0.12.1)
  thin (= 1.6.3)
  timeout (= 0.0.0)
  tinder (= 1.9.1)
  turbo-sprockets-rails3 (= 0.3.14)
  twitter (= 4.8.1)
  twitter-text (= 1.6.1)
  tzinfo (= 0.3.42)
  uglifier (= 2.5.3)
  user-agent (= 1.0.0)
  vcr
  will_paginate (= 3.0.3)
  yui-compressor (= 0.12.0)

BUNDLED WITH
   1.11.2
``

**Comment by TimMoore** [comment]
If you can help come up with a minimal Gemfile that reproduces the problem (one which people outside your org can test with) it will greatly help us speed up figuring out what's happening. Thanks!

**Comment by jnixon22** [comment]
Should have done it in the first place, but narrowed it down to 

```
source 'https://rubygems.org'

gem 'resque-scheduler', '2.2.0'
```

```
GEM
  remote: https://rubygems.org/
  specs:
    multi_json (1.11.2)
    rack (1.4.5)
    rack-protection (1.5.3)
      rack
    redis (3.0.4)
    redis-namespace (1.5.2)
      redis (~> 3.0, >= 3.0.4)
    resque (1.23.0)
      multi_json (~> 1.0)
      redis-namespace (~> 1.0)
      sinatra (>= 0.9.2)
      vegas (~> 0.1.2)
    resque-scheduler (2.2.0)
      redis (>= 3.0.0)
      resque (>= 1.20.0, < 1.25)
      rufus-scheduler (~> 2.0)
    rufus-scheduler (2.0.24)
      tzinfo (>= 0.3.22)
    sinatra (1.4.6)
      rack (~> 1.4)
      rack-protection (~> 1.4)
      tilt (>= 1.3, < 3)
    tilt (1.4.1)
    tzinfo (0.3.42)
    vegas (0.1.11)
      rack (>= 1.0.0)

PLATFORMS
  ruby

DEPENDENCIES
  resque-scheduler (= 2.2.0)

BUNDLED WITH
   1.11.2
```

**Comment by TimMoore** [comment]
Awesome, thanks! I can reproduce that.

**Comment by TimMoore** [comment]
It looks like this is similar to #3963 in that it's likely to be caused by an invalid `gemspec` in one of your dependencies.

That version of `resque-scheduler` has a bug in the `gemspec` that looks like it was fixed in v2.3.1: https://github.com/resque/resque-scheduler/commit/2f8d32bf779a5f09569cbda066b72f80ced58e86

So if you can update that gem to 2.3.1 or later, that's the best way to proceed. I'll leave this bug open for now so that we can make Bundler handle this case more gracefully, but at the end of the day there's only so much you can do with an invalid `gemspec` in a dependency so I think at best we would make it give a more informative error message that points directly to the problem gem.

**Comment by segiddins** [comment]
Closed by https://github.com/bundler/bundler/pull/4283.
